### ISSUE TITLE
[Suggestion] 

### ISSUE THREAD

**Issue by Mawobi** [FabioZumbi12/RedProtect#732]
**Is your feature request related to a problem? Please describe.**
I would like to allow players at End-Spawn to break and place every block exepting from Endstone and Obsidian.

**Describe the solution you'd like**
Theres an Admin-Flag for **allow-place** and **allow-break**. It would be really useful if you could add the **deny-place** and **deny-break** flag.

Greetings
~Mawobi

**Comment by FabioZumbi12** [comment]
Theres an option for this only on globalflags file, not as region flag.

**Comment by CloudeLecaw** [comment]
i agree with FabioZumbi12 usually can define this in the global flag,

The only problem i see here is that if i understood the OT korrekt he dont want this for the full world.
he want it just for his Region at the end where he defined his spawn in the endworld.
outside of this region this should be possible.
So he asks for "opposite"new flags workflow of allow-place  allow-break  that would be   **deny-place**  and **deny-break**
when useing /rp flag deny-place or /rp flag deny-break a GUI should open like we have it for the flags allow-place and allow-break.
If blocks are defined in deny-place or deny-break (so GUI is NOT empty) then it means players that are no member of the region could break/place everything except the blocks defined for deny-place and deny-break.

**Comment by Mawobi** [comment]
I absolutely agree you CloudeLecaw.
It would be helpful for example to deny breaking and placing Obsidian and Endstone in a farming-world-end only at the middle island. This would prevent griefing this floating island and would make it possible for many players to kill the enderdragon.
